core/templates/pages/terms-page/terms-page.component.html

```diff
@@ -68,10 +68,9 @@ <h2>Hosted Created Content and Intellectual Property</h2>
     <p>
       All Educational Content that is licensed to us pursuant to these Terms is
       then immediately licensed by us under the Creative Commons CC-BY-SA 4.0
-      license with a waiver of the attribution (BY) requirement, which means
-      that all of the content on Oppia is irrevocably licensed to the general
-      public and can be freely used, reused, and shared. You can read more about
-      the CC-BY-SA 4.0 license
+      license, which means that all of the content on Oppia is irrevocably
+      licensed to the general public and can be freely used, reused, and
+      shared. You can read more about the CC-BY-SA 4.0 license
```

Comment:
Hi @dchen97, can you please check if the wording here looks right?

Reply:
LGTM